index.html

```diff
@@ -691,9 +691,13 @@
 
 <div class="practice">
 
-<span class="practicelab" id="principle-identity-per-context">[=User agents=] should support
-their [=users=]' [=autonomy=] by helping them present their intended [=identity=] to each
-[=context=] that they visit.</span>
+<span class="practicelab" id="principle-identity-per-context">[=User agents=] should help
+their users present the [=identity=] those people want in each [=context=] they find
```

Comment:
I used "context" here instead of "situation" in the hope that https://github.com/w3ctag/privacy-principles/issues/110 will make it the right word.